README.md

```diff
@@ -83,19 +83,19 @@ mod test {
   #[test]
   fn test_set_temperature_20() {
       let mut scenario = Scenario::new();
-      let cond = scenario.create_mock_for::<AirConditioner>();
+      let mut cond = scenario.create_mock_for::<AirConditioner>();
 
       scenario.expect(cond.get_temperature_call().and_return(16));
       scenario.expect(cond.make_hotter_call(4).and_return(()));
 
-      set_temperature_20(&cond);
+      set_temperature_20(&mut cond);
   }
 }
 ```
 
 Run tests:
 
-```
+```rust
```

Comment:
That's true. At least not all of it. ;-)
I personally like the formatting better, but I can remove the tag if you like.